my-project/templates/product_category/show.html.twig

```diff
@@ -14,9 +14,11 @@
 {% block body %}
     <h1>ProductCategory</h1>
     {% for image in product_category.images %}
-        <img class="img-thumbnail img-250" src="{{ asset('uploads/images/' ~ image.path) }}">
+        <div class="img-container">
+            <div class="img-close-x btn btn-sm btn-danger-opacity" onclick="RemoveImageFromCategory('{{ path('remove_image_from_category', {'image': image.id, 'category': product_category.id}) }}', this.parentElement)">X</div>
```

Comment:
@mSobonVirtua is there a confirmation box like: Do you really want to remove this image? If not we would need one.